python/kiss_icp/tools/visualizer.py

```diff
@@ -20,223 +20,301 @@
 # LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
-import copy
 import importlib
 import os
 from abc import ABC
-from functools import partial
-from typing import Callable, List
 
 import numpy as np
 
-CYAN = np.array([0.24, 0.898, 1])
-RED = np.array([128, 0, 0]) / 255.0
-BLACK = np.array([0, 0, 0]) / 255.0
-BLUE = np.array([0.4, 0.5, 0.9])
-GRAY = np.array([0.4, 0.4, 0.4])
-SPHERE_SIZE = 0.20
+from kiss_icp.config.parser import KISSConfig
+
+BACKGROUND_COLOR = [0.8470, 0.8588, 0.8863]
```

Comment:
I'm not a big fan of discussing colors, mainly due to my condition. But I would probably pick the color palette we had before, just to make it look consistent with the ROS visualizations, that we make them look the same in https://github.com/PRBonn/kiss-icp/pull/285